<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TSPWindow</class>
 <widget class="QMainWindow" name="TSPWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>669</width>
    <height>479</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>TSPWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="font">
    <font>
     <family>MS Shell Dlg 2</family>
    </font>
   </property>
   <layout class="QGridLayout" name="gridLayout_4">
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item row="0" column="0">
     <widget class="QTabWidget" name="tabWidget">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="tabTags">
       <attribute name="title">
        <string>Тэги</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout">
        <property name="leftMargin">
         <number>0</number>
        </property>
        <property name="topMargin">
         <number>0</number>
        </property>
        <property name="rightMargin">
         <number>0</number>
        </property>
        <property name="bottomMargin">
         <number>0</number>
        </property>
        <item row="0" column="0">
         <widget class="TSPTableView" name="tableTags">
          <property name="sortingEnabled">
           <bool>true</bool>
          </property>
          <attribute name="horizontalHeaderCascadingSectionResizes">
           <bool>false</bool>
          </attribute>
          <attribute name="horizontalHeaderMinimumSectionSize">
           <number>25</number>
          </attribute>
          <attribute name="horizontalHeaderDefaultSectionSize">
           <number>80</number>
          </attribute>
          <attribute name="horizontalHeaderStretchLastSection">
           <bool>true</bool>
          </attribute>
          <attribute name="verticalHeaderVisible">
           <bool>false</bool>
          </attribute>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tabGroups">
       <attribute name="title">
        <string>Группы</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_5">
        <property name="leftMargin">
         <number>0</number>
        </property>
        <property name="topMargin">
         <number>0</number>
        </property>
        <property name="rightMargin">
         <number>0</number>
        </property>
        <property name="bottomMargin">
         <number>0</number>
        </property>
        <item row="0" column="0">
         <widget class="TSPTableView" name="tableGroups">
          <property name="sortingEnabled">
           <bool>true</bool>
          </property>
          <property name="cornerButtonEnabled">
           <bool>true</bool>
          </property>
          <attribute name="horizontalHeaderCascadingSectionResizes">
           <bool>false</bool>
          </attribute>
          <attribute name="horizontalHeaderMinimumSectionSize">
           <number>25</number>
          </attribute>
          <attribute name="horizontalHeaderDefaultSectionSize">
           <number>80</number>
          </attribute>
          <attribute name="horizontalHeaderStretchLastSection">
           <bool>true</bool>
          </attribute>
          <attribute name="verticalHeaderVisible">
           <bool>false</bool>
          </attribute>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tabDrivers">
       <attribute name="title">
        <string>Драйверы</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_2">
        <property name="leftMargin">
         <number>0</number>
        </property>
        <property name="topMargin">
         <number>0</number>
        </property>
        <property name="rightMargin">
         <number>0</number>
        </property>
        <property name="bottomMargin">
         <number>0</number>
        </property>
        <item row="0" column="0">
         <widget class="TSPTableView" name="tableDrivers">
          <property name="sortingEnabled">
           <bool>true</bool>
          </property>
          <attribute name="horizontalHeaderCascadingSectionResizes">
           <bool>false</bool>
          </attribute>
          <attribute name="horizontalHeaderMinimumSectionSize">
           <number>25</number>
          </attribute>
          <attribute name="horizontalHeaderDefaultSectionSize">
           <number>80</number>
          </attribute>
          <attribute name="horizontalHeaderStretchLastSection">
           <bool>true</bool>
          </attribute>
          <attribute name="verticalHeaderVisible">
           <bool>false</bool>
          </attribute>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar">
   <property name="enabled">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>669</width>
     <height>20</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_help">
    <property name="title">
     <string>Справка</string>
    </property>
    <addaction name="action_TSP"/>
   </widget>
   <widget class="QMenu" name="menu_window">
    <property name="title">
     <string>Окно</string>
    </property>
    <addaction name="action_refresh"/>
    <addaction name="action_resize"/>
    <addaction name="action_edit"/>
    <addaction name="action_addRows"/>
    <addaction name="action_fill"/>
    <addaction name="action_inc"/>
   </widget>
   <widget class="QMenu" name="menu_file">
    <property name="title">
     <string>Файл</string>
    </property>
    <addaction name="action_saveCopy"/>
   </widget>
   <widget class="QMenu" name="menu_view">
    <property name="title">
     <string>Вид</string>
    </property>
    <addaction name="action_showAccess"/>
   </widget>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>Тэг</string>
    </property>
    <addaction name="actionTrue"/>
    <addaction name="actionFalse"/>
   </widget>
   <addaction name="menu_file"/>
   <addaction name="menu_view"/>
   <addaction name="menu_window"/>
   <addaction name="menu_help"/>
   <addaction name="menu"/>
  </widget>
  <action name="actionTrue">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="rsrc.qrc">
     <normaloff>:/rsrc/indicator_true.png</normaloff>:/rsrc/indicator_true.png</iconset>
   </property>
   <property name="text">
    <string>Выставить true</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+1</string>
   </property>
  </action>
  <action name="actionFalse">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="rsrc.qrc">
     <normaloff>:/rsrc/indicator_empty.png</normaloff>:/rsrc/indicator_empty.png</iconset>
   </property>
   <property name="text">
    <string>Выставить false</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+0</string>
   </property>
  </action>
  <action name="actionConnect">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Включить</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+E</string>
   </property>
  </action>
  <action name="actionDisconnect">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Отключить</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+R</string>
   </property>
  </action>
  <action name="action_TSP">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>О компоненте TSP</string>
   </property>
   <property name="shortcut">
    <string>F1</string>
   </property>
  </action>
  <action name="action_error">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Узнать ошибку</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+H</string>
   </property>
  </action>
  <action name="action_refresh">
   <property name="text">
    <string>Обновить</string>
   </property>
   <property name="shortcut">
    <string>F5</string>
   </property>
  </action>
  <action name="action_resize">
   <property name="text">
    <string>Выровнять</string>
   </property>
  </action>
  <action name="action_edit">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Разрешить редактирование</string>
   </property>
  </action>
  <action name="action_addRows">
   <property name="text">
    <string>Добавить строки</string>
   </property>
  </action>
  <action name="action_saveCopy">
   <property name="text">
    <string>Сохранить копию</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+S</string>
   </property>
  </action>
  <action name="action_showAccess">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Отображать тип доступа</string>
   </property>
  </action>
  <action name="action_fill">
   <property name="text">
    <string>Проставить значения</string>
   </property>
  </action>
  <action name="action_inc">
   <property name="text">
    <string>Проставить инкремент</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>TSPTableView</class>
   <extends>QTableView</extends>
   <header>guiItems.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="rsrc.qrc"/>
 </resources>
 <connections/>
</ui>
